dnf-plugins-extras.spec

```diff
@@ -156,6 +164,37 @@ Obsoletes:	dnf-plugins-extras-migrate <= 0.0.4-2
 Migrate Plugin for DNF, Python 2 version. igrates yum's history, group and
 yumdb data to dnf.
 
+%package -n python-dnf-plugins-extras-kickstart
+Summary:	Kickstart Plugin for DNF
+Requires:	python-dnf-plugins-extras-common = %{version}-%{release}
+%if 0%{?fedora} >= 23
+BuildRequires:   python-kickstart
+%else
+BuildRequires:   pykickstart
+%endif
+%if 0%{?fedora} < 23
+Provides:	dnf-command(kickstart)
+Provides:	dnf-plugins-extras-kickstart = %{version}-%{release}
+%endif
+
+%description -n python-dnf-plugins-extras-kickstart
+Kickstart Plugin for DNF, Python 2 version. Install packages listed in a
+Kickstart file.
+
+%if 0%{?fedora} >= 23
+%package -n python3-dnf-plugins-extras-kickstart
```

Comment:
I forgot to push the commit that disables the plugin for python3 by default. It should be in now.